.changeset/unlucky-months-sort.md

```diff
@@ -0,0 +1,24 @@
+---
+'xstate': major
+---
+
+Strict mode is no longer available. You can no longer set `strict: true` to control:
```

Comment:
This was already removed [here](https://github.com/statelyai/xstate/pull/3455/files#diff-3f1cb738974f23da45b9ccda1df60076a239709f9354e23ad8bae87c44aef2c1). I think that the changeset added in this PR should be removed (as this PR only clean up some leftover from the other ones). The existing changeset doesn't have a lot of information so if you feel like the information added here is important u can add it to that other one.

Reply:
Removed